index.html

```diff
@@ -126,11 +129,41 @@ <h1 style="font-size: 65px;">
         destination
       </h1>
       <h4>
-        <a href="#about" class="btn-get-started scrollto">Download Now</a>
+        <a href="#about" class="btn-get-started scrollto" id="btn">Download Now</a>
       </h4>
     </div>
-    <img class="sec-img" src="images/rd.png" alt="" />
+    <img class="sec-img" src="images/rd.png" alt="" id="sec-img" style="padding: 0 0 0 0;"/>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.4/gsap.min.js"></script>
+<script>
+  const img = document.getElementById('sec-img');

```

Comment:
Same here too..we can make code reusable

Reply:
sorry i didn't get u

Reply:
@varshith257 that is added by mistakely , that code should be added in tilt effect PR.

I removed it now